pkg/controllers/nodeclaim/disruption/drift.go

```diff
@@ -47,6 +47,7 @@ type Drift struct {
 	instanceTypeNotFoundCheckCache *cache.Cache
 }
 
+// This function should only modify the nodeclaim's Status section
```

Comment:
nit: probably an unnecessary comment -- seems fine to just add the change -- it makes logical sense

Reply:
FWIW: If someone were to update the spec or metadata today anyways, it wouldn't propagate so them updating other things would require them to have to know to call Patch differently anyways